compiler/rustc_middle/src/query/mod.rs

```diff
@@ -204,6 +204,16 @@ rustc_queries! {
         desc { "getting the resolver for lowering" }
     }
 
+    /// Named module children from all kinds of items, including imports.
+    /// In addition to regular items this list also includes struct and variant constructors, and
+    /// items inside `extern {}` blocks because all of them introduce names into parent module.
+    ///
+    /// Module here is understood in name resolution sense - it can be a `mod` item,
+    /// or a crate root, or an enum, or a trait.
+    query module_children_local(key: LocalDefId) -> &'tcx [ModChild] {
```

Comment:
The `eval_always` comment https://github.com/rust-lang/rust/pull/114669#discussion_r2172640369 applies to this PR as well.